lib/graph.py

```diff
@@ -34,6 +34,11 @@ def escape(string):
         return string
 
 
+def wrap_command(limit, break_with, command):
+    occurrences = re.findall('.{1,%i}' % limit, command)
```

Comment:
I think it would be better to use [`textwrap.wrap`](https://docs.python.org/3/library/textwrap.html#textwrap.wrap) or the class version, as it is more likely to end the line on a hyphen or whitespace